WooCommerce/Classes/Authentication/AuthenticatedWebViewController.swift

```diff
@@ -4,9 +4,9 @@ import WebKit
 
 /// The web view to handle plugin setup in the login flow.
```

Comment:
Nit: maybe this comment should be updated too?

Reply:
Oops! Thanks, will do